SwiftBuddiesIOS/Targets/FeedModule/Sources/Components/ActionButton.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ActionButton.swift
+//  Feed
+//
+//  Created by Kate Kashko on 15.04.2024.
+//  Copyright © 2024 SwiftBuddies. All rights reserved.
+//
+
+import SwiftUI
+
+struct ActionButton: View {
+    let action: () -> Void
+    let imageName: String
+    let label: String
+    let isToggled: Bool
+    let toggledColor: Color
+
```

Comment:
```suggestion
  
    init(imageSystemName: String, label: String, isToggled: Bool, toggledColor: Color, action: @escaping () -> Void) {
        self.action = action
        self.imageSystemName = imageSystemName
        self.label = label
        self.isToggled = isToggled
        self.toggledColor = toggledColor
    }
```

completion/action olan yerlerde initializer yazmaya ozen gosterelim. Genel olarak initializer yazmak el aliskanligi olsun, aksi halde proje buyudugunde compiler'in kafasi karisabiliyor.